package.json

```diff
@@ -44,5 +44,8 @@
     "jest": "^24.0.0",
     "prettier": "^1.16.0",
     "rewire": "^4.0.1"
+  },
+  "dependencies": {
+    "acorn": "^7.1.0"
   }
```

Comment:
`acorn` is a parser so it shouldn't be in dependencies (should be in devDependencies).